src/webui/components/Icon/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @prettier
+ * @flow
+ */
+
+import React from 'react';
+import type { Node } from 'react';
+
+import capitalize from '../../utils/capitalize';
+import { SVG, Img, ImgWrapper } from './styles';
+import { IProps, IIconsMap } from './types';
+
+import brazil from './img/brazil.svg';
+import china from './img/china.svg';
+import india from './img/india.svg';
+import nicaragua from './img/nicaragua.svg';
+import pakistan from './img/pakistan.svg';
+import austria from './img/austria.svg';
+
+import spain from './img/spain.svg';
+import earth from './img/earth.svg';
+import verdaccio from './img/verdaccio.svg';
+
+export const Icons: $Shape<IIconsMap> = {
+  // flags
+  brazil,
+  spain,
+  china,
+  nicaragua,
+  pakistan,
+  india,
+  austria,
+
```

Comment:
Typo.. a new line ❌

Reply:
hahaha I wanted to separate the flags...I will remove it..